pkgs/applications/networking/seahub/default.nix

```diff
@@ -4,34 +4,29 @@
 , python3
 , makeWrapper
 , nixosTests
+, seafile-server-override ? null
 }:
 let
-  python = python3.override {
-    packageOverrides = self: super: {
-      django = super.django_3;
+  python = python3.override (lib.optionalAttrs (!builtins.isNull seafile-server-override) {
+    packageOverrides = final: prev: {
+      seaserv = prev.toPythonModule (seafile-server-override.override {
+        python3 = prev.python;
+      });
```

Comment:
```suggestion
```

this is a bit complex to account for overrides here to me

Reply:
This was a workaround for `makePythonPath` ignoring the module if it wasn't part of the python package set. I'll leave this open because the current workaround isn't great either.

Reply:
Nevermind, I just forgot to apply `toPythonModule`.